### DISCUSSION TITLE
Adding HTTP status codes responses

### DISCUSSION THREAD

**Discussion by BlackBucketCL** [microsoft/typespec#2907] [General]
Hello,

I'm new to Typespec and I don't know how to assign different status codes in the response. Let's say I need to send a 409 response or any other specific HTTP status code. 

Can you help me with that please? 

Thanks!

**Comment by bterlson** [comment]
You can see examples [here](https://typespec.io/docs/getting-started/getting-started-http#status-codes). Essentially, when an operation returns a model, that model is checked for a field with the `@statusCode` decorator, which becomes the status code for that operation.

**Reply by BlackBucketCL** [reply]
Thank you! working fine
